packages/api/src/main/java/com/incubator/edupayroll/entity/DocumentRow.java

```diff
@@ -0,0 +1,31 @@
+package com.incubator.edupayroll.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Entity
+@Table(name = "DocumentRow")
+public class DocumentRow extends BaseEntity {
```

Comment:
bu document row dediğimiz arkadaş bir de index: int field'ı ekler misin? tablodaki yerini bulmak için